mycore-pi/src/main/java/org/mycore/pi/MCRPIRegistrationService.java

```diff
@@ -27,6 +25,8 @@
 import org.mycore.pi.backend.MCRPI_;
 import org.mycore.pi.exceptions.MCRPersistentIdentifierException;
 
+import static org.mycore.access.MCRAccessManager.PERMISSION_WRITE;
```

Comment:
please put static imports before regular imports (like in eclipse) to keep format the same